server/src/main/java/org/apache/iotdb/db/query/control/QueryFileManager.java

```diff
@@ -119,6 +121,9 @@ void addFilePathToMap(long queryId, TsFileResource tsFile, boolean isClosed) {
     if (!pathMap.get(queryId).contains(tsFile)) {
       pathMap.get(queryId).add(tsFile);
       FileReaderManager.getInstance().increaseFileReaderReference(tsFile, isClosed);
+      logger.warn("{} is read-locked by {}", tsFile, queryId, new Exception());
     }
   }
+
+  private static final Logger logger = LoggerFactory.getLogger(QueryFileManager.class);

```

Comment:
may put this to the first line of Class will be better.

Reply:
I added it just to print some logs in tests for debugging, and I will remove it.